src/AutomationFramework/Config.java

```diff
@@ -0,0 +1,85 @@
+package AutomationFramework;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class Config {
+	public String FireFoxExePath = "C:\\Downloaded Things\\geckodriver.exe";
+	public String ChromeExePath = "C:\\Downloaded Things\\chromedriver.exe";
+	//public String WebsiteURL = "https://california.demo.collaborativefusion.com/";
```

Comment:
I think that in general its better to delete lines of code rather than commenting them out.

Reply:
will delete the comments in final copy of the code